### PR TITLE
[ReadTheDocs] Add style guide to navigation bar

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Adds the style guide to the Read The Docs sidebar.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -24,4 +24,5 @@ nav:
     - Developers: 
         - 'Overview': 'wiki/99 - Developers/README.md'
         - 'SQL Dictionary': 'wiki/99 - Developers/SQL Dictionary.md'
+        - 'Documentation Style Guide': 'HelpStyleGuide.md'
```
Comment (driusan) [inline mkdocs.yml]:
```suggestion
        - 'Help Style Guide': 'HelpStyleGuide.md'
```

The style guide only applies to help text.

Comment (johnsaigle) [reply mkdocs.yml]:
```suggestion
        - 'Style Guide (for Module help text)': 'HelpStyleGuide.md'
```

I think this is a bit more clear (though maybe wordy). Thoughts?

Comment (driusan) [reply mkdocs.yml]: you don't need the word "Module" in the parenthesis (but it might still be wordy if you remove it)

**Comment by johnsaigle** [comment]
Screen Shot 2019-12-17 at 13 29 48

**Pull request merged**